libs/base/Data/Strings.idr

```diff
@@ -199,6 +199,10 @@ export partial
 strTail : String -> String
 strTail = prim__strTail
 
+export
+total isNullStr : String -> Bool
+isNullStr = (== "")
```

Comment:
What differs this from `null`?

Reply:
For String isn't a member of Foldable interface. I was attempt to overload it to null letter if possible.